docs/src/main/sphinx/object-storage/metastores.md

```diff
@@ -500,6 +500,9 @@ following properties:
 * - `iceberg.rest-catalog.vended-credentials-enabled`
   - Use credentials provided by the REST backend for file system access.
     Defaults to `false`.
+* - `iceberg.rest-catalog.nested-namespace-support-enabled`
```

Comment:
Rename to `iceberg.rest-catalog.nested-namespace-enabled`.

Reply:
renamed